org.eclipse.jdt.core/search/org/eclipse/jdt/internal/core/search/indexing/IndexManager.java

```diff
@@ -1800,7 +1800,7 @@ public boolean execute(IProgressMonitor progress) {
 					if (JobManager.VERBOSE) {
 						Util.verbose("-> failed to update meta index for index " + indexFile.getName() //$NON-NLS-1$
 								+ " because of the following exception:"); //$NON-NLS-1$
-						e.printStackTrace();
+						Util.log(e);
```

Comment:
This is strange one - no idea why it expects a cause from IOException, is a single IOException not good enough?

Same as above, in this PR either don't touch code functionality inside "debug" or "verbose" blocks, or refactor to log some of the exceptions.